certificate-policy.md

```diff
@@ -1264,15 +1276,28 @@ Subscriber Certificates issued for delegated OCSP responders SHALL have a Validi
 
 ### 6.5.1 Specific computer security technical requirements
 
-The CA SHALL enforce multi-factor authentication for all accounts capable of directly causing certificate issuance.
-
-For all CAs operating under this policy, the computer security functions listed below are required. These functions may be provided by the operating system, or through a combination of operating system, software, and physical safeguards. The CA and its ancillary parts shall include the following functionality:  
+Administrator privileges to all Certificate System components SHALL only be granted to the Administrator trusted role. The CA SHALL enforce multi-factor authentication using a unique credential created by or assigned to the trusted role for all accounts capable of directly causing certificate issuance or authenticating to Certificate Systems. Multifactor authentication SHALL be implemented on all Certificate System components. If multifactor authentication is not supported or is not technically feasible to implement, the following username and password controls, where technically feasible, SHALL be implemented:
+
+- For accounts that are not publicly accessible (accessible only within a Secure or High Security Zone), require passwords have at leaast twelve (12) characters.
```

Comment:
It sounds like we should make the language more specific that any trusted role with operating system privileged access are required to have MFA to the operating system. Is the Fed 2FA policy CSIP?

A couple factors to consider with this (assuming one factor is PIV):
1) Issuance time of a PIV. If it take weeks or months to get a PIV, that has real contract impact. Alternative is use a locally issued token until a PIV is issued or use a locally issued token.
2) Availability of the infrastructure. If there was an availability issue in retrieving revocation data, would that impact PIV logon. (example of the PIV logon to the issuing CA system that does not have network connectivity.
3) Offline systems. PIV can be deployed without a domain controller.